src/global/Components/QrScanner/QrScanner.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import Webcam from 'react-webcam';
+import { QRCode } from 'jsqr';
+import getPixels from 'get-pixels';
+/* eslint-disable */
```

Comment:
why we disable eslint here?

Reply:
Cause of 
@typescript-eslint/ban-ts-ignore
Changed to disable-next-line